File: testdata/jennies/builders/dataquery_variant_builder/JavaBuilder/dataquery_variant_builder/LokiBuilderBuilder.java
package dataquery_variant_builder;


public class LokiBuilderBuilder implements cog.Builder<cog.variants.Dataquery> {
    protected final Loki internal;
    
    public LokiBuilderBuilder() {
        this.internal = new Loki();
    }
    public LokiBuilderBuilder expr(String expr) {
        this.internal.expr = expr;
        return this;
    }
    public Loki build() {
        return this.internal;
    }
}

File: testdata/jennies/builders/dataquery_variant_builder/CSharpBuilder/src/Grafana/Foundation/DataqueryVariantBuilder/LokiBuilderBuilder.cs
namespace Grafana.Foundation.DataqueryVariantBuilder;


public class LokiBuilderBuilder : Cog.IBuilder<Cog.Variants.Dataquery>
{
    protected readonly Loki @internal;

    public LokiBuilderBuilder()
    {
        this.@internal = new Loki();
    }

    public LokiBuilderBuilder Expr(string expr)
    {
        this.@internal.Expr = expr;
        return this;
    }

    public Loki Build()
    {
        return this.@internal;
    }
}
